src/gg.regression.unity.bots/Editor/Scripts/BotManagement/IRGBotFinder.cs

```diff
@@ -0,0 +1,71 @@
+using UnityEditor;
+using UnityEditor.Callbacks;
+using UnityEngine;
+using System.Linq;
+using System.Collections.Generic;
+
+public class IRGBotFinder : Editor
+{
+    [DidReloadScripts]
+    private static void OnScriptsReloaded()
+    {
+        // Only run this method in the Unity editor, not at runtime
+        if (!EditorApplication.isPlayingOrWillChangePlaymode || EditorApplication.isPlaying)
+        {
+            return;
+        }
+
+        // Find the IRGBotList ScriptableObject in your project
+        IRGBotList botList = LoadBotList();
+
+        if (botList == null)
+        {
+            Debug.LogWarning("IRGBotList ScriptableObject not found");
+            return;
+        }
```

Comment:
Instead, should this be created for them? This was actually one question I had from the video - how does this get into their project?

Reply:
+1